packages/strapi-plugin-upload/models/File.settings.json

```diff
@@ -50,6 +50,10 @@
       "type": "string",
       "configurable": false
     },
+    "resource_type": {
```

Comment:
This field won't make sense for a lot of other plugins. It's too specific to cloudinary. 

We could add a json metadata field to handle those kind of necessary informations for multiple other plugins for example

Reply:
Good idea. I'll make an update for that soon.

Reply:
@alexandrebodin It looks like the `public_id` field is also Cloudinary specific. Should we also move this field into the metadata field (calling it `provider_metadata`)?

Reply:
sure we can move it in that field

Reply:
Might not be a good idea anyway as it will remove the `public_id` data from previously uploaded images, so it won't be possible to delete those images after updating

Reply:
Yep :) Will just give details on the changelog and communicaition once we release this breaking change. but it's for the best